dbt_project.yml

```diff
@@ -15,6 +15,10 @@ vars:
     ad_report_hourly: "{{ ref('stg_tiktok_ads__ad_report_hourly') }}"
     ad_history: "{{ ref('stg_tiktok_ads__ad_history') }}"
     advertiser: "{{ ref('stg_tiktok_ads__advertiser') }}"
+    ad_country_report: "{{ ref('stg_tiktok_ads__ad_country_report') }}"
```

Comment:
This appears to no longer being built, so it can be removed.

Reply:
Correct, this was initially included and then removed. Removing the artifact.